02-household-queries/chainlit-household-bot.py

```diff
@@ -72,6 +98,12 @@ async def on_click_settings(action: cl.Action):
     E.g. **bold**, *italic*, `code`, [links](https://www.example.com), etc.
 """
 
+@cl.action_callback("resetDB")
+async def on_click_resetDB(action: cl.Action):
+    persistent_client = cl.user_session.get("persistent_client")
+    if persistent_client is None:
+        persistent_client = chromadb.PersistentClient(settings=Settings(allow_reset=True), )
```

Comment:
Can you do the following to avoid creating the `chroma` folder?
```suggestion
        persistent_client = chromadb.PersistentClient(path="./chroma_db", settings=Settings(allow_reset=True), )
```

Reply:
Why is this different than line 209
https://github.com/navapbc/labs-gen-ai-experiments/pull/8/files#diff-7c8e574c10cfec8dafa0ca000decfbaa6cf0d032423f4fee72b0d1a48dd736f5R209